src/main/java/org/javarosa/xpath/expr/XPathJsonPropertyFunc.java

```diff
@@ -0,0 +1,48 @@
+package org.javarosa.xpath.expr;
+
+import org.json.JSONObject;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.DataInstance;
+import org.json.JSONException;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+/** Utility for hidden values as geocoder receivers
+ *
+ * @author rcostello
+ * @return A String value for the property name passed in if that property exists else a blank String
+ */
+
+public class XPathJsonPropertyFunc extends XPathFuncExpr {
+    public static final String NAME = "getJsonProperty";
```

Comment:
We should stick to the 'dash-case' naming convention. I'm also suggesting that we drop the 'get' (similar to 'selected-at')

```suggestion
    public static final String NAME = "json-property";
```